woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php

```diff
@@ -19,7 +19,12 @@
 function ebanx_update_converted_value () {
 	$gateway = new WC_EBANX_Gateway();
```

Comment:
I'm so glad this is never called!
A good fatal error for instantiating an abstract class would be in order.